internal/schemautil/userconfig/handle.go

```diff
@@ -219,9 +234,15 @@ func handleAggregateTypeProperty(
 ) (map[string]*jen.Statement, error) {
 	r := make(map[string]*jen.Statement)
 
+	req := map[string]struct{}{}
+
+	if sreq, ok := p["required"].([]interface{}); ok {
+		req = SliceToKeyedMap(sreq)
+	}
+
 	switch at {
 	case "object":
-		v, err := handleObjectProperty(n, p, t)
+		v, err := handleObjectProperty(n, p, t, req)
```

Comment:
If `handleObjectProperty` handles just one property, then it is possible to this:
```go
requiredList, ok := p["required"].([]interface{})

// This is mandatory
if !ok {
    return fmt.Errorf("...")
}

handleObjectProperty(a, b, c, slices.Contains(requiredList, fieldName)
```

Reply:
it handles multiple properties in downstream call of `convertPropertiesToSchemaMap`